lib/uploads/common.js

```diff
@@ -245,9 +245,9 @@ var DEF_PREFIX_LEN = 3;
 var MIN_PREFIX_LEN = 1;
 var MAX_PREFIX_LEN = 4;
 
-// Range of allowed part numbers is [0, 10000).
+// Range of allowed part numbers is [0, 800).
 var MIN_PART_NUM = 0;
-var MAX_PART_NUM = 9999;
+var MAX_PART_NUM = 799;
```

Comment:
Do you think we should change:

```
var PART_NUM_REGEX = /^([0-9]|[1-9][0-9]{0,2})$/;
```

there I changed `s/,3/,2/` because the number of valid digits is lower now.

Reply:
Sure.